docs/references/protocol/transactions/types/mptokenissuanceset.md

```diff
@@ -39,4 +40,17 @@ Transactions of the `MPTokenIssuanceSet` type support additional values in the `
 | `tfMPTLock`        | `0x00000001` | 1             | Enable to lock balances of this MPT issuance. |
 | `tfMPTUnlock`      | `0x00000002` | 2             | Enable to unlock balances of this MPT issuance. |
 
+## Error Cases
+
+Besides errors that can occur for all transactions, {% $frontmatter.seo.title %} transactions can result in the following [transaction result codes](../transaction-results/index.md):
+
+| Error Code                | Description |
+|:--------------------------|:------------|
+| `temDISABLED`             | The `MPTokensV1` amendment is disabled. You will also receive this error if you include a `DomainID` field in the transaction, but the `PermissionedDomains` and `SingleAssetVault` amendments are both disabled. |
+| `tecNO_DELEGATE_PERMISSION` | You are attempting to use a delegate account with insufficient permissions to submit this transaction. |
```

Comment:
I think this code is not specific to MPTokenIssuanceSet, and it's been replaced with a `ter` code in the next release also (relating to the bug that is the reason for `PermissionDelegationV1_1`)